samples/webserver/index.js

```diff
@@ -29,6 +31,42 @@ onload = function() {
 
   var logToScreen = function(log) {
     logger.textContent += log + "\n";
+    logger.scrollTop = logger.scrollHeight;
+  }
+
+  var destroySocketById = function(socketId) {
+    tcpSocket.disconnect(socketId,function(){
+      tcpSocket.close(socketId);
+    });
+  }
+
+  var closeServerSocket = function() {
+    if(serverSocketId) {
+      tcpServer.close(serverSocketId);
+    }
+
+    tcpServer.onAccept.removeListener(onAccept);
+    tcpSocket.onReceive.removeListener(onReceive);
+  }
+
+  var sendReplyToSocket = function(socketId,buffer,keepAlive) {
+    // verify that socket is still connected before trying to send data
+    tcpSocket.getInfo(socketId,function(socketInfo) {
+      if(socketInfo.connected !== true) {
```

Comment:
`if (!socketInfo.connected) {` seems easier to read than `if(socketInfo.connected !== true) {`